wiki/People/User_group/en.md

```diff
@@ -0,0 +1,27 @@
+---
+tags:
+  - usergroup
+---
+
+# User group
+
+*Notice: The Chat Bot and Announce user group listings are not public.*
```

Comment:
what's the difference between a `Note` and a `Notice`?

Reply:
![image](https://user-images.githubusercontent.com/36758269/227051208-13b76fa3-73f6-48e8-95a3-ca166909b3de.png)

should be `Note` i suppose

```suggestion
*Note: The Chat Bot and Announce user group listings are not public.*
```